app/contracts/queries/projects/project_queries/base_contract.rb

```diff
@@ -57,10 +57,10 @@ def user_is_logged_in
 
     def allowed_to_modify_private_query
```

Comment:
I second guessed whether using `#model.editable?` here as it's quite broad but would be consistent with the rest of checks in other places.